src/Spice86/Behaviors/UseParentListBoxContextMenuBehavior.cs

```diff
@@ -0,0 +1,60 @@
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Interactivity;
+
+namespace Spice86.Behaviors {
```

Comment:
Any reason to use namespace block?

Reply:
Guess I forgot to tell the AI about that standard :)
Fixed.